### PR TITLE
PMM-12348 Restart clickhouse after adding flag.

### PR DESCRIPTION
**Pull request by BupycHuk**
PMM-12348

Link to the Feature Build: SUBMODULES-3523

If this PR adds or removes or alters one or more API endpoints, please review and add or update the relevant [API documents](https://github.com/percona/pmm/tree/main/docs/api) as well:

- [ ] API Docs updated

If this PR is related to some other PRs in this or other repositories, please provide links to those PRs:

- Links to related pull requests (optional).

### CODE REVIEW

**Review by idoqo — APPROVED** [review]

**Review by JiriCtvrtka — APPROVED** [review]

File: update/ansible/playbook/tasks/roles/initialization/tasks/main.yml [hunk 1]
```diff
@@ -71,21 +71,41 @@
...
+          file:
+            path: "/srv/clickhouse/flags"
+            state: directory
+            owner: root
+            group: pmm
+            recurse: true
+
+        - name: Create empty file to convert clickhouse databases from ordinary to atomic
+          file:
+            path: "/srv/clickhouse/flags/convert_ordinary_to_atomic"
```
Comment (ademidoff) [inline update/ansible/playbook/tasks/roles/initialization/tasks/main.yml:92]: Why would we create it again, if that file already exists? I'm looking at line 108.

Comment (BupycHuk) [reply update/ansible/playbook/tasks/roles/initialization/tasks/main.yml:92]: good point

File: update/ansible/playbook/tasks/roles/initialization/tasks/main.yml [hunk 2]
```diff
@@ -71,21 +71,41 @@
...
+          command: supervisorctl restart clickhouse
+          become: true
+          changed_when: true
+
+        - name: Restart clickhouse EL9
+          when: (ansible_distribution == 'OracleLinux' or ansible_distribution == 'AlmaLinux') and ansible_distribution_major_version == '9' and is_docker
+          command: /usr/local/bin/supervisorctl restart clickhouse
+          become: true
+          changed_when: true
+      when: is_database_ordinary.stat.exists
```
Comment (BupycHuk) [inline update/ansible/playbook/tasks/roles/initialization/tasks/main.yml:108]:
```suggestion
      when: not is_database_ordinary.stat.exists
```

File: update/ansible/playbook/tasks/roles/initialization/tasks/main.yml [hunk 3]
```diff
@@ -71,21 +71,41 @@
...
-        group: pmm
-        recurse: true
-
-    - name: Create empty file to convert clickhouse databases from ordinary to atomic
-      file:
-        path: "/srv/clickhouse/flags/convert_ordinary_to_atomic"
-        state: touch
-        owner: root
-        group: pmm
+    - name: Check that Clickhouse database engine is ordinary
```
Comment (BupycHuk) [inline update/ansible/playbook/tasks/roles/initialization/tasks/main.yml:74]:
```suggestion
    - name: Check that conversion flag isn't created
```

**Review by ademidoff — APPROVED** [review]

**Pull request merged**